### PR TITLE
Improved clean script

### PR DESCRIPTION
**Pull request by codejamninja**
Fixes issue #307

### CODE REVIEW

**Comment by cryptobioz** [comment]
Seems great! Could you please keep the entry `rm -f bivac` and add your command above it?

**Comment by codejamninja** [comment]
I can, but `rm -r bivac` will be unnecessary when #306 is accepted.

**Comment by codejamninja** [comment]
Just added it

**Comment by cryptobioz** [comment]
`make bivac` build the binary, it makes sense that `make clean` removes the binary.

File: Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ vet: main.go
 	go vet $<
 
 clean:
-	rm -f bivac
+	git clean -fXd -e \!vendor -e \!vendor/**/* && rm -f ./bivac
```
Comment (cryptobioz) [inline Makefile]: Why putting the two commands on the same line? Wouldn't be better for code maintainability to split this into two different lines?

**Pull request merged**